ethernet/w5500_spi/w5500_spi.c

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include "pico/stdlib.h"
+#include "pico/binary_info.h"
+#include "hardware/spi.h"
+#include "w5500.h"
+#include "wizchip_conf.h"
+#include "loopback.h"
+
+
+/* Example code to talk to a wiznet MEMS accelerometer and gyroscope.
```

Comment:
This this comment is  a hangover from  a previous example?

Reply:
Ah I forgot to push the changes I made to that file into this branch. I'll commit them now